roslib/Header header
string timeline
string[] variable_names
string[] variable_values
int32[] variable_types

int32 TYPE_UNKNOWN = 0
int32 TYPE_SINGLETON = 1 
int32 TYPE_INTERVAL = 2
int32 TYPE_ENUMERATED = 3





